torch/_torch_docs.py

```diff
@@ -2821,18 +2823,21 @@ def merge_dicts(*dicts):
     tensor(1.9073e-06)
 """.format(**common_args))
 
-add_docstr(torch.isinf,
-           r"""
+add_docstr(torch.isinf, r"""
 isinf(input) -> Tensor
 
-Returns a new tensor with boolean elements representing if each element is `+/-INF` or not.
-Complex values are infinite when their real and/or imaginary part is infinite.
+Tests if each element of :attr:`input` is infinite
```

Comment:
You can follow the template for other comparison ops here, 
```
Returns a new tensor with boolean elements representing if each element is `infinite` (equal to positive or negative infinity) or not.
```

Reply:
I was actually trying to explicitly avoid that templating because I thought the language of "Returns a new tensor..." was out of place with other language in our docs that describes the computation rather than the return type in the summary.

Reply:
Then should other ops (comparison, isfinite etc) be updated too?

Reply:
Yes. This PR is just to get the language right before changing all of them.